### PR TITLE
Improve mysql instance label values.

### PR DESCRIPTION
**Pull request by wez470**
I thought it made more sense that the MySQL StatefulSet and Service had instance values that corresponded to the name. Please correct me if I'm wrong.

This is my first open source contribution so please tell me if there is anything else I need to do. Should I update in other languages as well?

### CODE REVIEW

File: content/en/docs/concepts/overview/working-with-objects/common-labels.md [hunk 1]
```diff
@@ -170,4 +170,4 @@ metadata:
 
 With the MySQL `StatefulSet` and `Service` you'll notice information about both MySQL and Wordpress, the broader application, are included.
 
-{{% /capture %}}
\ No newline at end of file
+{{% /capture %}}
```
Comment (wez470) [inline content/en/docs/concepts/overview/working-with-objects/common-labels.md]: Not sure what changed here. I just edited the instance values with the github UI.

**Comment by sftim** [comment]
Thanks for the pull request @wez470 

Assuming I've understood the intent and the outcome correctly:
/lgtm

**Pull request merged**